cmake/linux/package_linux.sh.in

```diff
@@ -0,0 +1,170 @@
+	STRIP="-no-strip"
+fi
+
+# Console colors
+RED="\\x1B[1;31m"
+GREEN="\\x1B[1;32m"
+YELLOW="\\x1B[1;33m"
+PLAIN="\\x1B[0m"
+
+function error {
+	echo -e "   ${PLAIN}[${RED}error${PLAIN}] ${1}"
+	return 1
+}
+
+function success {
+	echo -e "   ${PLAIN}[${GREEN}success${PLAIN}] ${1}"
+}
+
+function skipped {
+	echo -e "   ${PLAIN}[${YELLOW}skipped${PLAIN}] ${1}"
+}
+
+
+# Check for problematic install locations
+INSTALL=$(echo "@CMAKE_INSTALL_PREFIX@" | sed 's/\/*$//g')
+if [ "$INSTALL" == "/usr/local" ] || [ "$INSTALL" == "/usr" ] ; then
+	error "Incompatible CMAKE_INSTALL_PREFIX for creating AppImage: @CMAKE_INSTALL_PREFIX@"
+fi
+
+echo -e "\nWriting verbose output to \"${LOGFILE}\""
+
+# Ensure linuxdeployqt uses the same qmake version as cmake
+export PATH="$(dirname "@QT_QMAKE_EXECUTABLE@")":$PATH
+
+# Fetch portable linuxdeployqt if cache is older than $DAYSOLD
+echo -e "\nDownloading linuxdeployqt to ${LINUXDEPLOYQT}..."
+mkdir -p "$HOME/bin"
+DAYSOLD=2
+if env -i which linuxdeployqt > /dev/null 2>&1; then
+	skipped "System already provides this utility"
+elif ! find "$LINUXDEPLOYQT" -mtime -$DAYSOLD 2>/dev/null|grep -q "." > /dev/null 2>&1; then
+	url="https://github.com/probonopd/linuxdeployqt/releases/download/continuous/linuxdeployqt-continuous-$(uname -p).AppImage"
+	echo "   [.......] Couldn't find linuxdeployqt newer than $DAYSOLD days old"
+	echo "   [.......] Downloading ($(uname -p)): ${url}"
+	wget "$url" -O "$LINUXDEPLOYQT" -q || (rm "$LINUXDEPLOYQT" && false)
+	chmod +x "$LINUXDEPLOYQT"
+	touch "$LINUXDEPLOYQT"
+	success "Downloaded $LINUXDEPLOYQT"
+	"$LINUXDEPLOYQT" --appimage-extract > /dev/null 2>&1
+	mv "squashfs-root/usr/bin/appimagetool" "$APPIMAGETOOL"
+	success "Extracted $APPIMAGETOOL"
+	mv "squashfs-root/usr/bin/mksquashfs" "$USERBIN/mksquashfs"
+	success "Extracted $USERBIN/mksquashfs"
+	rm -rf "squashfs-root/"
+
+else
+	skipped "$LINUXDEPLOYQT is less than $DAYSOLD days old"
+fi
+
+# Make skeleton AppDir
+echo -e "\nCreating ${APPDIR}..."
+rm -rf "${APPDIR}"
+mkdir -p "${APPDIR}usr"
+success "Created ${APPDIR}"
+
+# Clone install to AppDir
+echo -e "\nCopying @CMAKE_INSTALL_PREFIX@ to ${APPDIR}..."
+cp -R "@CMAKE_INSTALL_PREFIX@/." "${APPDIR}usr"
+rm -rf "${APPDIR}usr/include"
+success "${APPDIR}"
+
+# Copy rawwaves directory for stk/mallets
+mkdir -p "${APPDIR}usr/share/stk/"
+cp -R /usr/share/stk/rawwaves/ "${APPDIR}usr/share/stk/"
+
+# Create a wrapper script which calls the lmms executable
+mv "${APPDIR}usr/bin/lmms" "${APPDIR}usr/bin/lmms.real"
+# shellcheck disable=SC1083
+cat >"${APPDIR}usr/bin/lmms" <<EOL
+#!/usr/bin/env bash
+DIR="\$( cd "\$( dirname "\${BASH_SOURCE[0]}" )" && pwd )"
+export LD_LIBRARY_PATH=\$DIR/usr/lib/:\$DIR/usr/lib/lmms:\$LD_LIBRARY_PATH
+# Prevent segfault on VirualBox
+if lsmod |grep vboxguest > /dev/null 2>&1; then
+   echo "VirtualBox detected.  Forcing libgl software rendering."
+   export LIBGL_ALWAYS_SOFTWARE=1;
+fi
+QT_X11_NO_NATIVE_MENUBAR=1 \$DIR/usr/bin/lmms.real "\$@"
```

Comment:
Add `QT_AUTO_SCREEN_SCALE_FACTOR=1` to force scaling (Qt 5.6+ only)

Reply:
@flynn16 Added to the checklist, thanks!